Assets/Outline_Controller.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Outline_Controller : MonoBehaviour
+{
+    private MeshRenderer renderer;
+
+    public float maxOutlineWidth;
+
+    public Color OutlineColor;
+
+
+    private void Start()
+    {
+        renderer = GetComponent<MeshRenderer>();
+    }
+
+    public void ShowOutline()
+    {
+
+        foreach (var mat in renderer.materials)
```

Comment:
It looks like this is the Show/Hide outline that seems to be in the other functions, could you replace those places with just calling this instead?